extensions/community_extensions/dsa_transparency.md

```diff
@@ -0,0 +1,379 @@
+</tr>
+<tr>
+<td><code>required</code></td>
+<td>integer</td>
+<td>Flag to indicate if DSA information should be made available. This will signal if the bid request belongs to an Online Platform/VLOP, such that a buyer should respond with DSA Transparency information based on the pubrender value.
+    <p><code>0</code> = Not required</p>
+    <p><code>1</code> = Supported, bid responses with or without DSA object will be accepted</p>
+    <p><code>2</code> = Required, bid responses without DSA object will not be accepted</p>
+    <p><code>3</code> = Required, bid responses without DSA object will not be accepted, Publisher is an Online Platform</p></td>
+</tr>
+<tr>
+<td><code>pubrender</code></td>
+<td>integer</td>
+<td>Flag to indicate if the publisher will render the DSA Transparency info. This will signal if the publisher is able to and intends to render an icon or other appropriate user-facing symbol and display the DSA transparency info to the end user.
+    <p><code>0</code> = Publisher can't render</p>
+    <p><code>1</code> = Publisher could render depending on adrender</p>
+    <p><code>2</code> = Publisher will render</p></td>
+</tr>
+<tr>
+<td><code>datatopub</code></td>
+<td>integer</td>
+<td>Independent of pubrender, the publisher may need the transparency data for audit purposes.
+    <p><code>0</code> = do not send transparency data</p>
+    <p><code>1</code> = optional to send transparency data</p>
+    <p><code>2</code> = send transparency data</p></td>
+</tr>
+<tr>
+<td><code>transparency</code></td>
+<td>array of object</td>
+<td>Array of objects of the entities that applied user parameters and the parameters they applied.</td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Object: Transparency&nbsp;</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>domain</code></td>
+<td>string&nbsp;</td>
+<td>Domain of the entity that applied user parameters&nbsp;</td>
+</tr>
+<tr>
+<td><code>params</code></td>
+<td>array of integer</td>
+<td>Array for platform or sell-side use of any user parameters (using <a href="#user_parameters" target="_blank" rel="noopener">the list provided by DSA Transparency Taskforce</a>). <em>Note; See definition and list of possible </em><a href="#user_parameters" target="_blank" rel="noopener"><em>user parameters as listed here</em></a><em>, applied consistently in both bid request and/or bid response.</em></td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Sample OpenRTB 2.6 Bid Request with DSA transparency:</h4>
+
+```
+{
+    "id": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+    "at": 1,
+    "cur": [
+        "USD"
+    ],
+    "regs": {
+        "ext": [{
+            "dsa": {
+                "required": 3, 
+                "pubrender": 0,
+                "datatopub": 2,
+                "transparency": [{
+                    "domain": "platform1domain.com",
+                    "params": [1]},
```

Comment:
Same as in the comment for the response, either `params` or `userparameters`, as long as it's the same for both